raiden/ui/app.py

```diff
@@ -121,7 +121,7 @@ def _setup_matrix(config):
 
             light_client_transports.append(light_client_transport)
 
-        hub_transport = MatrixTransport(config["transport"]["matrix"])
+        hub_transport = MatrixTransport(config["address"], config["transport"]["matrix"])
```

Comment:
there are other places in the code in which the `MatrixTransport` class is instantiated, but it does not receive 2 parameters.

search for `MatrixTransport(` and fix these too

Reply:
Yes i know, thanks